src/index.html

```diff
@@ -1,13 +1,351 @@
 <!DOCTYPE html>
-<html lang="en">
+<html class="page" lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <title>MyBiKE</title>
+    <link rel="icon" href="images/favicon.png">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="page__section header">
+      <div class="container">
+        <div class="header__content">
+          <div class="header__top">
+            <a href="#" class="logo">
+              <img
+                class="logo__image"
+                src="images/MyBiKE.svg"
+                alt="MyBiKE"
+              >
+            </a>
+
+            <a href="tel:+1 234 5555-55-55" class="icon icon--phone">
+              <div class="icon__tooltip">+1 234 5555-55-55</div>
+            </a>
+
+            <a href="#menu" class="icon icon--menu"></a>
+          </div>
+          <h1 class="header__title">Take the Streets</h1>
+        </div>
+      </div>
+    </header>
+
+    <nav class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="menu__content">
+          <div class="menu__top">
+            <a href="/" class="logo">
```

Comment:
It transfer you to 404 error page
```suggestion
            <a href="#" class="logo">
```